runtime/syntax/mom.vim

```diff
@@ -0,0 +1,16 @@
+" VIM syntax file
+" Language:	nroff
+" Maintainer:	John Marshall <jmarshall@hey.com>
```

Comment:
again, a different maintainer

Reply:
Same as above, it was included in nroff and its maintainer showed interest in including MOM syntax, so I thought most prudent to leave the existing nroff maintainer intact and copy headers;
it's just a suggestion as both maintainers expressed interest in special treatment for MOM files as rather special kind of NROFF files